src/main/java/org/entando/kubernetes/controller/coordinator/EntandoControllerCoordinator.java

```diff
@@ -58,6 +58,8 @@ public EntandoControllerCoordinator(KubernetesClient client) {
     }
 
     public void onStartup(@Observes StartupEvent event) {
+        //TODO extract TLS and CA certs and write them to the standard secret names
```

Comment:
is this TODO to be solved in this ticket or is for the future?

Reply:
It is some instructions for future improvements